website/index.html

```diff
@@ -211,10 +211,28 @@ <h4 id="bonus">Bonus</h4>
             <p>Add a static import for <code>ButterKnife.findById</code> and enjoy even more fun.</p>
 
             <h3 id="download">Download</h3>
-            <h4>Gradle</h4>
-            <pre class="prettyprint">compile 'com.jakewharton:butterknife:<span class="version"><em>(insert latest version)</em></span>'
-apt 'com.jakewharton:butterknife-compiler:<span class="version"><em>(insert latest version)</em></span>'</pre>
+            Configure your project-level build.gradle to include the 'android-apt' plugin:
+            <h4>Project-level build.gradle</h4>
+            <pre class="prettyprint">
+buildscript {
+  repositories {
+    mavenCentral()
+   }
+  dependencies {
+    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+  }
+}
+            </pre>
+            Then, apply the 'android-apt' plugin in your module-level build.gradle and add the Butter Knife dependencies:
+            <h4>Module-level build.gradle</h4>
+            <pre class="prettyprint">
+apply plugin: 'android-apt'
 
+dependencies {
+  compile 'com.jakewharton:butterknife:<span class="version"><em>(insert latest version)</em></span>'
```

Comment:
There is actually a piece of Javascript which loads the latest version automatically, but this is not working on GH pages, so hence better to add a fixed version.